packages/spacecat-shared-data-access/src/models/configuration.js

```diff
@@ -24,6 +25,10 @@ const Configuration = (data = {}) => {
     state.handlers = state.handlers || {};
     state.handlers[type] = { ...handlerData };
   };
+  self.getSlackRolesByAuditType = (type) => {
```

Comment:
slack roles are not related to "audit types" - did you mean something like "getSlackRoleMembersByRole"?

Reply:
changed